Server/Hambasafe.Api/Startup.cs

```diff
@@ -85,6 +82,17 @@ public void Configure(IApplicationBuilder app, IHostingEnvironment env, ILoggerF
             loggerFactory.AddConsole(Configuration.GetSection("Logging"));
             loggerFactory.AddDebug();
 
+            if (env.IsDevelopment())
+            {
+                app.UseDeveloperExceptionPage();
+
+                using (var serviceScope = app.ApplicationServices.GetRequiredService<IServiceScopeFactory>().CreateScope())
+                {
+                    serviceScope.ServiceProvider.GetService<HambasafeDataContext>().Database.Migrate();
+                    //serviceScope.ServiceProvider.GetService<ApplicationDbContext>().Database.Migrate();
```

Comment:
Only create and migrate HambasafeDataContext schema for now.  ApplicationDbContext can be added when we need it.  This allows us to get up and running in your local environment relatively painlessly.